doc/user-guide/data_model.rst

```diff
@@ -614,19 +619,18 @@ an array of the same name, you must specify the ``preference``:
    >>> ugrid.active_scalars_info
 
 This can also be set when plotting using the ``preference``
-parameter in :func:`add_mesh() <pyvista.Plotter.add_mesh>` or
-:func:`pyvista.plot`.
```

Comment:
`pyvista.plot()` doesn't support `preference` right now. But it probably should.

Reply:
It does because it passes extra kwargs to `Plotter.add_mesh`.  The following works:

```py
import pyvista as pv

mesh = pv.Sphere()
mesh['data'] = range(mesh.n_cells)
mesh['data'] = range(mesh.n_points)

mesh.plot(preference='cells')
mesh.plot(preference='points')
```

Reply:
Aaah, I should've known that! Thanks, reverting.